scenarios/kubernetes_bazel.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs bazel build/test for current repo."""
+
+import argparse
+import os
+import subprocess
+import sys
+
+ORIG_CWD = os.getcwd()
+
+def test_infra(*paths):
+    """Return path relative to root of test-infra repo."""
+    return os.path.join(ORIG_CWD, os.path.dirname(__file__), '..', *paths)
+
+def check(*cmd):
+    """Log and run the command, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    subprocess.check_call(cmd)
+
+
+def echo_result(res):
+    """echo error message bazed on value of res"""
+    echo_map = {
+        0:'Success',
+        1:'Build failed',
+        2:'Bad environment or flags',
+        3:'Build passed, tests failed or timed out',
+        4:'Build passed, no tests found',
+        5:'Interrupted'
+    }
+    print echo_map.get(res, 'Unknown exit code : %s' % res)
+
+
+BUILD_MAP = {
```

Comment:
I see. This will make us maintain paths from multiple places? How do we enforce people update the ci job and k/k Makefile at the same time?